llm/pdf-processing/pdf-processing-api.py

```diff
@@ -0,0 +1,77 @@
+import os
+from collections.abc import Iterator
+from dotenv import load_dotenv
+
+from datachain import DataChain, C, File, DataModel
+
+from unstructured.partition.api import partition_via_api
+from unstructured.cleaners.core import clean
+from unstructured.cleaners.core import replace_unicode_quotes
+from unstructured.cleaners.core import group_broken_paragraphs
+from unstructured.embed.huggingface import (
+    HuggingFaceEmbeddingConfig,
+    HuggingFaceEmbeddingEncoder,
+)
+
+load_dotenv()
+
+
+# Define the output as a DataModel class
+class Chunk(DataModel):
+    key: str
+    text: str
+    embeddings: list[float]
+
+
+# Define embedding encoder
+
+embedding_encoder = HuggingFaceEmbeddingEncoder(config=HuggingFaceEmbeddingConfig())
```

Comment:
btw, minor question - do you have to pass an empty `HuggingFaceEmbeddingConfig`?